### DISCUSSION TITLE
How to change row color in datagrid?

### DISCUSSION THREAD

**Discussion by Seartur** [AvaloniaCommunity/Material.Avalonia#292] [Q&A] [answered]
Hi. Why can't I change the color in row in datagrid?

`<DataGrid Height="1200" Width="1000"  x:Name="AlarmsDG" RowBackground="BlueViolet" AutoGenerateColumns="true"></DataGrid>`

How does "RowBackground" work?

**Comment by SKProCH** [comment, marked as answer]
Seems like this is a bug. You can track progress here: https://github.com/AvaloniaCommunity/Material.Avalonia/issues/293

**Reply by Seartur** [reply]
In 3.0.0.36-nightly work!
